src/components/Banner/BannerMobile.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { Dialog } from '@santiment-network/ui'
+import styles from './BannerMobile.module.scss'
+
+const classes = { title: styles.title }
+
+const BannerMobile = ({ children }) => {
+  return (
+    <Dialog trigger={<></>} position='bottom' defaultOpen classes={classes}>
```

Comment:
This is a case for a `Modal` with a render callback as children. `Dialog` is an entity which has a straight connection to some action, e.g. click on the "delete" button, and following response to it.

Reply:
hm, ok. You wrote that `Modal` is  low-level component, so I decided use `Dialog`) I'll change